### PR TITLE
Add unit (%) to table header

### PR DESCRIPTION
**Pull request by MicheleK17**
Ticket: #217
Change-Id: I7dd1dbabde90add6e8bf56d46188ec5fabb1957c

### CODE REVIEW

File: src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx [hunk 1]
```diff
@@ -53,28 +54,33 @@ const StyledTableCell = withStyles(() =>
...
         </StyledTableCell>
     );
-    headerValues.forEach((element): void => {
-        elements.push(
-            <StyledTableCell key={`header-${element}`} css={headerCellStyle}>
-                {element}
+    headerValues.forEach((headerValue): void => {
+        let headerTitle = headerValue
+        if (tableOption === "relative") {
+            headerTitle = `${headerValue} in %`
```
Comment (GerberI) [inline src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx:70]: I would use a translation here too, even if it is the same text for now.

**Review by GerberI — CHANGES_REQUESTED** [review]
Commit: Add unit (%) to table header

File: src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx (hunk 1 above)
Comment (MicheleK17) [reply src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx]: Done.

**Review by GerberI — APPROVED** [review]
Commi: Add unit translation
thx

**Comment by GerberI** [comment]
Can be merged to develop

**Pull request merged**